marlowe-dashboard-client/src/Component/ConfirmContractActionDialog/State.purs

```diff
@@ -62,7 +61,7 @@ component
   => MonadTime m
   => Toast m
   => H.Component CCAD.Query Input Msg m
-component = connect (selectEq _.currentTime) $
+component = connect (selectEq selectSlice) $
```

Comment:
Main changes to this component: Added `roleTokens` to entries pulled from the store, removed `contractUserParties` from derived state.